CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [3.0.1] - 2023-07-21
```

Comment:
mainline is no longer 3.0.1

mainline contains a beta fix so it has a higher minor version, I think